x-team/xp-ui
# Left-align menu items and display applicant initials

**Release Type:** *Non-Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3837

## Description

Left-align menu items within the HeaderBar

Add an `avatarName` property to the ProfileAvatar properties which can
be used to generate applicant initials in place of the old placeholder
icon. Also add this to containing properties.

## Checklist


## Related PRs

branch | PR
------ | ------
XP-3837-menu-tweaks | [XP Registration](https://github.com/x-team/xp-registration/pull/XP-3837-menu-tweaks)

## Steps to Test or Reproduce

See the new stories under `UI Components > Header Bar > use cases`.

Other test steps listed on the PR listed above.

## Impacted Areas in Application

XP Registration header bar

## Screenshots

See other PR.
